### PR TITLE
Added Docker

### PR DESCRIPTION
**Pull request by pandeyashu08**
### This PR setups:

- [x] Added `Dockerfile` and `Docker-Compose`
- [x] Added `Dockerignore`
- [x] Added `health-route` in  index.js

**Previous merge conflicts has been resolved.**

### Screenshots:
image

### CODE REVIEW

**Comment by pandeyashu08** [comment]
@Tony-ArtZ This this all the merge conflicts have been resolved, can you please review it

File: .env.example [hunk 1]
```diff
@@ -1,3 +1,11 @@
-MONGODB_URL= <YOUR_CONNECTINO_URL>
-PORT= <PORT-TO-RUN-SERVER>
-JWT_SECRET="<your-JWT_SECRET>"
+# MongoDB Configuration
+
+# Use this URL to set up MongoDB in Docker
+MONGODB_URL_Docker=mongodb://mongodb:27017/
```
Comment (dead8309) [inline .env.example:4]: Use `MONGODB_URL`only for env key

**Comment by pandeyashu08** [comment]
@dead8309 Sorry for changing the env key , I have fixed it now

**Comment by dead8309** [comment]
Remove duplicate mongodb url key too

**Comment by pandeyashu08** [comment]
Done

**Comment by dead8309** [comment]
Thanks for your contribution!!

**Pull request merged**